docs/pages/tutorials/templates_1.md

```diff
@@ -339,7 +346,9 @@ To be added.
 
 To be added.
 
-!!! details "Complete template YAML"
+:::{admonition} Complete template YAML
+:collapsible: closed
```

Comment:
Collapsing does not seem to work with our theme. Anyways, I thought this would be the admonition corresponding to Documenter's details.